impl/src/expand.rs

```diff
@@ -307,12 +309,12 @@ fn impl_enum(input: Enum) -> TokenStream {
                     let self_provide = if type_is_option(backtrace_field.ty) {
                         quote! {
                             if let ::core::option::Option::Some(backtrace) = backtrace {
-                                #request.provide_ref::<std::backtrace::Backtrace>(backtrace);
+                                #request.provide_ref::<::std::backtrace::Backtrace>(backtrace);
```

Comment:
MINOR: It would be nice if we gave a better compiler error if a user tried to specify a backtrace when we're not in `std` mode, which could advise the user to enable the `std` feature to use this?

I believe the right place for this would be in `valid.rs`.